version.props

```diff
@@ -1,8 +1,8 @@
 ﻿<Project>
   <PropertyGroup>
-    <VersionPrefix>2.1.2</VersionPrefix>
-    <VersionSuffix>rtm</VersionSuffix>
-    <PackageVersion Condition="'$(IsFinalBuild)' == 'true' AND '$(VersionSuffix)' == 'rtm' ">$(VersionPrefix)</PackageVersion>
+    <VersionPrefix>2.1.6</VersionPrefix>
+    <VersionSuffix>servicing</VersionSuffix>
```

Comment:
I guess we're doing this to follow what .NET does? Fine with me.

Is the line below the only other place we special case `rtm`?

Reply:
Yes to both questions.